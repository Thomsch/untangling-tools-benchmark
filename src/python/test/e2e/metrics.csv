Closure,103,5,3,8,8.75,70,0,0
Closure,98,5,1,12,8.25,99,0,1
Csv,8,3,1,5,4.8,24,0,1
Jsoup,24,3,1,3,6.0,18,0,0
Lang,8,3,1,8,5.375,43,2,1
