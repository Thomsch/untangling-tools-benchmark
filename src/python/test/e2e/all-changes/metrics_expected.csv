project,vid,files_updated,hunks,average_hunk_size,code_changed_lines,tangled_lines,tangled_hunks
Closure,103,2,4,3.0,12,0.0,0
Closure,103,12,8,4
Closure,103,70,978,1032
Csv,8,1,3,7.0,21,0.0,1
Csv,8,21,13,8
Csv,8,24,2140,2138
Jsoup,24,1,1,1.0,1,0.0,0
Jsoup,24,1,1,0
Jsoup,24,18,312,328
Lang,63,1,4,5.75,23,1.0,1
Lang,63,23,23,2
Lang,63,64,5767,5781
Lang,8,1,4,3.5,14,2.0,1
Lang,8,14,3,15
Lang,8,43,2310,2351
